Make Opening hours more consistent

jpidelatorre opened the pull request:
- Added OSMOpeningHours to PointFeatureMarker to show formatted open hours.

closes #469

leonardehrenfried commented:
![Screenshot from 2021-05-21 09-30-55](https://user-images.githubusercontent.com/151346/119099515-43d55280-ba17-11eb-983c-1d2a605f3d61.png)

This doesn't seem to work because the content has a different format. Can you just make text a bit smaller, please? No need to be perfect just a little less visually jarring.

jpidelatorre commented:
@leonardehrenfried I think it works most of the time.

![image](https://user-images.githubusercontent.com/104569/119128303-7ecd8b00-ba03-11eb-9771-c99bc4dfc618.png)

I'll make the text smaller too.

jpidelatorre commented:
It also works with bike infrastructure.

![image](https://user-images.githubusercontent.com/104569/119128457-acb2cf80-ba03-11eb-9736-80e63a516ba9.png)

jpidelatorre commented:
@leonardehrenfried The font size is now smaller. Let me know if I should reduce the size of it.

leonardehrenfried commented:
@jpidelatorre The problem is not the the font size of the OSMOpeningHours component but that the data is completely wrong as the content of the GeoJSON layer is different.

Compare:

![image](https://user-images.githubusercontent.com/151346/119487280-76ee4d80-bd59-11eb-8c78-4a9f78ed7a50.png)

with:

![Screenshot from 2021-05-25 13-04-53](https://user-images.githubusercontent.com/151346/119487535-cfbde600-bd59-11eb-9830-01235d6201d9.png)

So the format is simply not compatible with the component.

For this reason, please completely stop using OSMOpeningHours and simply *reduce the font of the existing text*. Please don't change *anything else*.

MartinH-open commented:
On the example of "Nette Toilette" this is strange:
Searching for "Nette Toilette" in all of Herrenberg in OSM reports the first instance near Würzburg.
None of the StadtNavi "Nette Toilette" ist visible in OSM (for me). I wonder where this comes from and if this is a reason for the reported problems here ?

leonardehrenfried commented:
You want to use the following query to find them: https://overpass-turbo.eu/s/17IJ

MartinH-open commented:
Thanks @leonardehrenfried 
I see now these are part of the attributes ( sorry - I did not know this )
In other areas these are added as extra entities "toilet" on the map  - perhaps some inconsistency in OSM 

```
Tags 15
addr:city=Herrenberg
addr:housenumber=1
addr:postcode=71083
addr:street=Marktplatz
changing_table=yes
name=Bürgeramt
office=government
opening_hours=Mo-We 08:30-17:00; Th 08:30-18:30; Fr 08:30-12:00; Sa 09:30-12:00
toilets=yes
toilets:female=yes
toilets:male=yes
toilets:scheme=Nette Toilette
toilets:wheelchair=yes
```

jpidelatorre commented on app/component/map/PointFeatureMarker.js:
```diff
@@ -179,7 +179,11 @@ const PointFeatureMarker = ({
         />
-        {popupContent && <div className="card-text">{popupContent}</div>}
+        {popupContent && (
+          <div className="location-popup-wrapper">
```
Added the same padding present in the opening hours table.

jpidelatorre commented:
@leonardehrenfried I changed back the table and reduced the text. I'm seeing 20 files changed, but I only modified 2. Was there any revert in the `next` branch?

The files I changed are:
```
app/component/map/PointFeatureMarker.js
app/component/map/popups/marker-popup.scss
```

leonardehrenfried commented:
No there wasn't. Please start a new branch, use `git cherry-pick` to move your commits to this branch, create a new PR and delete this one.
